delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/indexing/BlazegraphIndexingCoordinator.scala

```diff
@@ -84,6 +86,7 @@ private class IndexingStream(
   )(implicit sc: Scheduler): IO[Nothing, Stream[Task, Unit]] =
     for {
       _     <- client.createNamespace(namespace, indexProperties).hideErrorsWith(illegalArgument)
+      _     <- cache.remove(projectionId)
```

Comment:
wouldn't it be more like `cache.put(projectionId, initialProgress)` ?

Reply:
cache.put(projectionId, initialProgress) would only work if there's no value in the cache yet, otherwise when we restart the view, the initialProgress will be earlier than the current and the update will be ignored by the KeyValueStore

Reply:
then:
```
   _     <- cache.remove(projectionId)
   _     <- cache.put(projectionId, initialProgress)
```

Reply:
I think 
```scala
      _     <- cache.remove(projectionId)
      _     <- cache.put(projectionId, initialProgress)
```
would make most sense.